engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1557,8 +1557,9 @@ public boolean getExecuteInSequence(final HypervisorType hypervisorType) {
             case LXC:
                 return false;
             case VMware:
-                final Boolean fullClone = HypervisorGuru.VmwareFullClone.value();
-                return fullClone;
+                final Boolean fullClone = getFullCloneConfiguration();
+                final Boolean allowParallel = getAllowParallelExecutionConfiguration();
+                return fullClone && !allowParallel;
```

Comment:
@DaanHoogland @nvazquez @sureshanaparti @andrijapanicsb 

I have discussed with Daan about this part of code.
It looks in 4.17, the global configuration `VmwareFullClone` impacts not only `CopyCommand`, but all other vm commands like `StartCommand`, `StopCommand`, `MigrateCommand`, `RebootCommand`.
This seems to be wrong, in my opinion.

Reply:
@nvazquez @sureshanaparti @andrijapanicsb @rohityadavcloud any opinions on this? I think it is not harmfull but on the other hand formally wrong. I can see the point but feel that if we have no user asking for fine tuning this we shouldn´t bother. cc @weizhouapache

Reply:
see the new issue #6433 for further discussion if needed.

Reply:
I am not clear yet on this change's impact but it seems like a good improvement